com.ibm.wala.cast.java.test.data/src/javaonepointfive/MoreOverriddenGenerics.java

```diff
@@ -38,6 +38,7 @@
 package javaonepointfive;
 
 public class MoreOverriddenGenerics {
+	@SuppressWarnings("InnerClassMayBeStatic")
```

Comment:
Any way to just disable diagnostics on the whole module, rather than adding suppressions?  Not a huge deal, but since it's a source front end in principle even annotations could affect the behavior of analyses

Reply:
We can define a new IntelliJ IDEA file scope that contains just files used as test inputs. Then we can reduce the severity level of individual inspections to "No highlighting, only fix" for files in that "test inputs" scope. I think that will have the desired effect.

If it's OK with you, I prefer to take that on as a separate task after merging this pull request.